src/node.rs

```diff
@@ -407,6 +407,11 @@ impl Node {
             .map(P2pNode::public_id)
     }
 
+    /// Returns a set of elders we should be connected to.
```

Comment:
`/// Returns a set of elders we can communicate with`

At the end we won't necessarily be connected, so probably best to not add a comment that will soon be misleading.